package.json

```diff
@@ -4,10 +4,14 @@
   "description": "the mighty option parser used by yargs",
   "main": "index.js",
   "scripts": {
-    "fix": "standard --fix",
-    "test": "c8 --reporter=text --reporter=html  mocha test/*.js",
-    "posttest": "standard",
-    "coverage": "c8 report --check-coverage check-coverage --lines=100 --branches=97 --statements=100"
+    "fix": "standardx --fix && standardx --fix **/*.ts",
```

Comment:
depending on @mleguen's feedback, I might be tempted to use `gts` here.

Reply:
In a separate PR, perhaps?